src/gui/dragAndDrop.js

```diff
@@ -1,7 +1,49 @@
-function pickUpNode() {
-    return "TEST";
+/**
+ * Handles processing of drag and drop events for the mathematical Nodes
+ * @type {number}
```

Comment:
What is this? Dangling docstring?

Reply:
I have no idea.. haven't seen that before. I'll fix it when I get home from dinner

Reply:
Should be gone now